testing/it/e2e-selenium-utils/src/main/java/com/adobe/cq/wcm/core/components/it/seljup/util/components/teaser/v1/Teaser.java

```diff
@@ -27,6 +27,7 @@
 public class Teaser extends BaseComponent {
     private static String teaser = ".cmp-teaser";
     private static String teaserImage = ".cmp-teaser__image";
+    private static String teaserImageLink=".cmp-image__link";
```

Comment:
Please add spaces before and after the `=` sign

Reply:
done